npm/qsharp/ux/circuit-vis/utils.ts

```diff
@@ -11,6 +11,38 @@ import {
 import { ComponentGrid, Operation } from "./circuit";
 import { Register } from "./register";
 
+/**
+ * Performs a deep equality check between two objects or arrays.
+ * @param obj1 - The first object or array to compare.
+ * @param obj2 - The second object or array to compare.
+ * @returns True if the objects are deeply equal, false otherwise.
+ */
+export const deepEqual = (obj1: any, obj2: any): boolean => {
```

Comment:
```suggestion
export const deepEqual = (obj1: unknown, obj2: unknown): boolean => {
```

This would be a good use of `unknown` over `any`